src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -414,6 +423,7 @@ else if (entity instanceof EntityCitizen && colony.isValidAttackingGuard((Entity
 
     /**
      * Check if a position is within regular attack distance.
+     * TODO:Fix so it actually says if sth is within attack Dist.
```

Comment:
Future TODO or forgot to remove it?

Reply:
ye future todo, didnt wanna add it to this one

Reply:
Yeah, it's pretty big already.